neptune/new/exceptions.py

```diff
@@ -316,6 +316,11 @@ def __init__(self):
         super().__init__("Cannot synchronize offline runs without a project.")
 
 
+class NeedExistingRunForReadOnlyMode(NeptuneException):
+    def __init__(self):
+        super().__init__("An existing run must be provided when running in read-only mode.")
```

Comment:
Let's write all new exception messages in new pretty format. See `NeptuneRunResumeAndCustomIdCollision` for example.

Reply:
updated, thanks